docs/docs/deploy/index.md

```diff
@@ -57,7 +56,7 @@ This will ask you a few questions regarding your install, download the latest
 release for your platform, then create an administrator user and print to the
 console instructions for logging in to the web UI.
 
-![install complete](https://user-images.githubusercontent.com/52545545/171948328-4771552f-e5dd-4c30-8c0b-baac80b6e7b1.png){:width="600"}
+![install complete](https://user-images.githubusercontent.com/52545545/171948328-4771552f-e5dd-4c30-8c0b-baac80b6e7b1.png)
 
 **Note**: During install new firewall rules are added, which can interrupt the
```

Comment:
I believe this no longer applies as it was fixed in firezone's `0.4.5` release.

https://github.com/firezone/firezone/pull/749/files

Reply:
👍